[hipblas] rocblas.h instead of rocblas/rocblas.h

Attempting to install ROCm from the AUR without arch4edu and receiving a few build errors. Because the errors appear to not be linked, I'm creating two issues (#801). 

The first error is hopefully an easy fix, I'm receiving a build error for hipblas:
```
/home/whisper/.cache/paru/clone/hipblas/src/hipBLAS-rocm-5.1.3/library/src/hcc_detail/hipblas.cpp:7:10: fatal error: 'rocblas.h' file not found
#include "rocblas.h"
         ^~~~~~~~~~~
```

The [hipblas](https://github.com/ROCmSoftwarePlatform/hipBLAS/blob/develop/library/src/hcc_detail/hipblas.cpp) repository upstream appears to have this fixed, but the files from the AUR do not. 

In addition, is there a way to build a modified package and have it be accepted by the paru helper? Editing the file directly from the paru cache and executing `make install` in the build directory appears to work (build, install, and symlinks successful), but when attempting to install `rocm-hip-sdk`, I am still required to install hipblas. Apologies if this is a simple question.

--- a/hipblas/.SRCINFO
+++ b/hipblas/.SRCINFO
@@ -1,8 +1,8 @@
 pkgbase = hipblas
 	pkgdesc = ROCm BLAS marshalling library
-	pkgver = 5.1.3
+	pkgver = 5.2.0
 	pkgrel = 1
-	url = https://rocmdocs.amd.com/en/latest/ROCm_Libraries/ROCm_Libraries.html#hipblas
+	url = https://hipblas.readthedocs.io/en/latest/
 	arch = x86_64
 	license = MIT
 	makedepends = cmake
@@ -10,7 +10,9 @@ pkgbase = hipblas
 	depends = hip
 	depends = rocblas
 	depends = rocsolver
-	source = hipblas-5.1.3.tar.gz::https://github.com/ROCmSoftwarePlatform/hipBLAS/archive/rocm-5.1.3.tar.gz
-	sha256sums = f0fdaa851971b41b48ec2e7d640746fbd6f9f433da2020c5fd95c91a7473d9e1
+	source = hipblas-5.2.0.tar.gz::https://github.com/ROCmSoftwarePlatform/hipBLAS/archive/rocm-5.2.0.tar.gz
+	source = include-path.patch::https://github.com/ROCmSoftwarePlatform/hipBLAS/commit/84b593c7c6acf176b21841a0441801d5a254c62b.patch
+	sha256sums = 5e9091dc4ef83896f5c3bc5ade1cb5db8e1a6afc451dbba4da19d8a7ec2b6f29
+	sha256sums = 34652acc94caad9cf9b22fcf7658b7c7a3fb4c1b517b3aa15e5cc35e973b4fc6
 
 pkgname = hipblas

--- a/hipblas/PKGBUILD
+++ b/hipblas/PKGBUILD
@@ -1,22 +1,30 @@
 # Maintainer: Torsten Keßler <t dot kessler at posteo dot de>
 # Contributor: Markus Näther <naether.markus@gmail.com>
 pkgname=hipblas
-pkgver=5.1.3
+pkgver=5.2.0
 pkgrel=1
 pkgdesc='ROCm BLAS marshalling library'
 arch=('x86_64')
-url='https://rocmdocs.amd.com/en/latest/ROCm_Libraries/ROCm_Libraries.html#hipblas'
+url='https://hipblas.readthedocs.io/en/latest/'
 license=('MIT')
 depends=('hip' 'rocblas' 'rocsolver')
 makedepends=('cmake' 'gcc-fortran')
 _git='https://github.com/ROCmSoftwarePlatform/hipBLAS'
-source=("$pkgname-$pkgver.tar.gz::$_git/archive/rocm-$pkgver.tar.gz")
-sha256sums=('f0fdaa851971b41b48ec2e7d640746fbd6f9f433da2020c5fd95c91a7473d9e1')
+source=("$pkgname-$pkgver.tar.gz::$_git/archive/rocm-$pkgver.tar.gz"
+        "include-path.patch::https://github.com/ROCmSoftwarePlatform/hipBLAS/commit/84b593c7c6acf176b21841a0441801d5a254c62b.patch")
+sha256sums=('5e9091dc4ef83896f5c3bc5ade1cb5db8e1a6afc451dbba4da19d8a7ec2b6f29'
+            '34652acc94caad9cf9b22fcf7658b7c7a3fb4c1b517b3aa15e5cc35e973b4fc6')
 _dirname="$(basename "$_git")-$(basename "${source[0]}" ".tar.gz")"
+
+prepare() {
+    # See https://github.com/ROCmSoftwarePlatform/hipBLAS/commit/84b593c7c6acf176b21841a0441801d5a254c62b
+    cd "$_dirname"
+    patch -Np1 -i "$srcdir/include-path.patch"
+}
 
 build() {
   # -fcf-protection is not supported by HIP, see
-  # https://github.com/ROCm-Developer-Tools/HIP/blob/rocm-5.1.x/docs/markdown/clang_options.md
+  # https://docs.amd.com/bundle/ROCm-Compiler-Reference-Guide-v5.2/page/Appendix_A.html
   CXX=/opt/rocm/bin/hipcc \
   CXXFLAGS="${CXXFLAGS} -fcf-protection=none" \
   cmake -B build -Wno-dev \

